guides/common/modules/proc_enabling-structured-apt-content.adoc

```diff
@@ -0,0 +1,51 @@
+[id="enabling-structured-apt-content"]
+= Enabling structured APT content
+
+You can enable structured APT content on your {Project} to use the same Deb repository structure as the remote repositories they are synchronized from.
+
+.Prerequisites
+* The host is registered to {ProjectName}.
+include::snip_prerequisite-project-client-repository-enabled.adoc[]
+ifdef::orcharhino[]
+* Your {DL} host uses the latest {project-client-name}.
+For more information, see https://atixservice.zendesk.com/hc/de/articles/16326419356572[Deb content: Using structured APT] in _ATIX Service Portal_.
+endif::[]
+ifndef::orcharhino[]
+* Your {DL} host runs `subscription-manager` version 1.29.35-3 or higher.
+endif::[]
+
+.Procedure
+. Backup your {ProjectServer}.
++
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+. Verify that there are no running tasks:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} health check --label foreman-tasks-not-running
+----
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+. On the *Content* tab, set the *Enable structured APT for deb content* setting to `Yes`.
+. On your {ProjectServer}, start {Project} maintenance mode:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} maintenance-mode start
+----
+. Migrate your Deb content:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake katello:migrate_structure_content_for_deb
+----
++
+On {Project} with 40 Deb repositories published to many content view versions and promoted to multiple lifecycle environments each, this step took 15 minutes.
```

Comment:
This amount of  time may differ and depends on packages, repositories, the used environment and other tasks running on your machine.

Reply:
Slightly reworded and applied.